src/Subgraph.ts

```diff
@@ -2,7 +2,7 @@ import axios from 'axios'
 import { Order, OrderTypes } from './types/subgraph'
 import { type Chain, chains, Chains, ClientOptions } from './types/networks'
 import { Address } from 'viem'
-import { validateOrder } from './validations'
+import { validateOrder } from './subgraph-validations'
```

Comment:
I moved the subgraph response validations to a separate file. These are different, so we should check for strings mostly. We have only orders added here. Not fully tested yet